app/controllers/v0/preneeds/preneed_attachments_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module V0
+  module Preneeds
+    class PreneedAttachmentsController < PreneedsController
+      def create
+        preneed_attachment = ::Preneeds::PreneedAttachment.new
+        preneed_attachment.set_file_data!(params[:preneed_attachment][:file_data])
```

Comment:
Can this match what we're doing for pension/burials, as far as params being passed in and data being sent back? (It looks different, though I could be wrong)

Reply:
this controller matches the education benefits claim controller and structures the params in the standard rails way, the other controller is non standard and later we should change it to match this one